docs/components/buttons/Example.vue

```diff
@@ -11,6 +11,9 @@
     <w-button secondary quiet>Secondary Quiet</w-button>
     <w-button negative>Negative</w-button>
     <w-button negative quiet>Negative Quiet</w-button>
+    <w-button pill>Pill</w-button>
+    <w-button pill loading>Loading</w-button>
```

Comment:
I know we have examples like the ones above in [Warp React Storybook](https://warp-ds.github.io/react/?path=/story/buttons-button--example) but I think I heard that Button of "pill" variant was supposed to be used with icons only (and/or numbers?). If that's true, it might be better to update the examples to reflect that.

Reply:
Ah yes I have a vague recollection of that as well! Also, when looking at @adidick's [figma sketches](https://www.figma.com/file/oHBCzDdJxHQ6fmFLYWUltf/Adi-sandbox%3A-Warp-variables-v2.0?node-id=1308%3A38516&mode=dev), it looks like the examples of button with variant pill are only displayed with an icon: . So I can change the example if that should be the case, but I guess we should then also inform the users in our tech-docs that they should only use an icon and/or number when using button with variant pill? :)